### PR TITLE
Mapi 628 merge webhook api docs

### PR DESCRIPTION
**Pull request by edafeo1**
Merge all webhook content from https://webhooks15.docs.apiary.io/#reference and https://securewebhooks.docs.apiary.io/# to form one coherent documentation in MM master doc

### CODE REVIEW

File: spec/openapi.yaml [hunk 1]
```diff
@@ -3643,6 +3657,9 @@ paths:
           "template": {
             "id":"$mtId",
             "status":"$statusCode"}
+          "read_timeout": 5000,
+          "retries": 3,
+          "retry_delay": 30
```
Comment (paultirta) [inline spec/openapi.yaml:3662]: Can you add these properties to the sections below? Around line 3680

Comment (edafeo1) [reply spec/openapi.yaml:3662]: the section around 3680 is about type of events , in the other webhook documents it is not listed there

Comment (paultirta) [reply spec/openapi.yaml:3662]: Sorry, I mean the section before that. It should be added in the

> A valid webhook must consist of the following properties

**Review by paultirta — APPROVED** [review]

**Comment by loilet** [comment]
Hi @edafeo1 ,

I check your deployment did not include my merge request: #69 .

Could you please help check to resolve it?

Thank you!

**Pull request merged**